content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md

```diff
@@ -0,0 +1,184 @@
+
+<img src="https://camo.githubusercontent.com/fabf9ea48fe61d2d71e84f86772677d865f2023ce4d58ee148e90e9b42276b7f/68747470733a2f2f692e696d6775722e636f6d2f75415a326637502e6a7067" width="100%" alt="photo of a basic building block with one camera">
+
+
+
+*Image of the simple sience setup, a basic building block with one fixed camera, by Mik Schutte.*
+* Main project website: https://github.com/mik-schutte/SimpleScienceSetup
+* Project start: 2022
+* Core development team size: 2
+
+## Hardware products
+starter kit for composing experimental setups, created with CNC and 3D printed plastic
+
+## Hardware maturity
+These are prototypes, we still need to test how they will age.
+
+{{< /card >}}
+
+
+
+
+## Using Bpod.
+
+As stated, a big part of our project has been to transition our laboratory interface from CED and SPIKE2 to BPOD.
+The laboratory interface is used to coordinate the experiments in our case it determines what to do with licking input, when to give a microstimulation and when to start saving frames.
+BPOD is a finite state machine that operates through MatLab and has an active community of users.
+The community provides a nice platform for individual support.
+However, the use of BPOD has a small learning curve and coding your own paradigm can be difficult.
+Luckily there is a clear to understand [wiki page](https://sites.google.com/site/bpoddocumentation/home) that guides new users through the basics. 
+
+We tried to opt for [pyBPOD](https://pybpod.readthedocs.io/projects/pybpod-api/en/v1.8.1/) to remove the paywall that is accompanied with the MatLab software.
+Yet, the pyBPOD software and community are not as well developed as the original version.
+This made it difficult for a beginning user to code their own paradigm and led to the choice of the original version.
+
+
+
+
+# Successes and failures
+**Success :**
+Making the microstimulation project open source with regards to both soft- and hardware has already facilitated information transfer between colleagues. For instance, the build-instructions have been utilized to re-build the setup at the lab in my absence. Additionally, information is currently helping a collaboration with [the Gilad lab](https://medicine.ekmd.huji.ac.il/en/research/arielgi/Pages/aboutTheLab.aspx) located in Jerusalem. Finally, our analysis software and accompanying explaination has been used by students in the lab. 
+
+**Fails :**
+In the past we have had problems with the ordering of components. This led to long delays which, in turn, caused us to be less productive during a colaboration with the [Delft Open Hardware Academy](https://www.openhardware.academy/01_Welcome.html). We also have have lost some time while trying to develop the BPOD laboratory interface in Python. [PyBPOD](https://pybpod.readthedocs.io/projects/pybpod-api/en/v1.8.1/) is a suitable product, yet I would discourage it's use for new users.
+
```

Comment:
## I am a better researcher

The fellowship has been instrumental in shaping my career profile and career path. Through hands-on experience, I acquired valuable skills in soldering, PCB design, 3D printing, and project management. These technical skills have enhanced my competence as a scientist and are actively being put to use. For instance, I have recently designed a PCB layout for microstimulation; the Arduino configures the waveform and soldered wires send this to the stimulus isolator with everything nicely housed in its 3D-printed ‘containers’.  Moreover, the exposure to project management has honed my ability to plan, organize, and execute tasks efficiently. Overall, the fellowship has made me into a more well-rounded researcher and has feeling well-prepared for the coming years of my PhD.